docs/how-to-guides/develop-a-tool/add-a-tool-icon.md

```diff
@@ -0,0 +1,64 @@
+# Add a Tool Icon
+A tool icon serves as a graphical representation of your tool in the user interface (UI). Follow this guidance to add a custom tool icon when developing your own tool package.  
+
+Adding a custom tool icon is optional. If you do not provide one, the system uses a default icon.
+
+## Prerequisites
+
+- Create a tool project as described in [Create and Use Tool Package](create-and-use-tool-package.md).
+
+- Install the dependent library:
```

Comment:
Install dependencies of generating icon data URI.

This step should be after prepare a image.

Reply:
Changed, thanks.